server/db/schema/05_quotas.sql

```diff
@@ -5,7 +5,7 @@ CREATE TABLE quotas
   id                SERIAL    PRIMARY KEY,
 
   user_id           INTEGER   REFERENCES users(id) ON DELETE CASCADE,
-  time_allotment    INTERVAL  NOT NULL default '2 hours',
+  time_allotment    INTERVAL  NOT NULL default '3 hours',
```

Comment:
Let's lower the default, 3 hours is a lot of time. I suggest 1.5 hours.

Reply:
Done.